app/frontend/src/application/jobseekers/manageQualifications.test.js

```diff
@@ -1,36 +1,37 @@
 import manageQualifications, {
-  addDeletionEventListener,
   addEventListenerForAddSubject,
   rowMarkup,
   addSubject,
-  insertDeleteButton,
+  renumberRow,
+  renumberRows,
   onDelete,
   DELETE_BUTTON_CLASSNAME,
   FIELDSET_CLASSNAME,
+  GOVUK_INPUT_CLASSNAME,
   ROW_CLASS,
-  SUBJECT_LINK_ID, renumberRemainingRows,
+  SUBJECT_LINK_ID,
 } from './manageQualifications';
 
 describe('manageQualifications', () => {
   const originalMarkup = `<div class="${FIELDSET_CLASSNAME}">
       <div class="${ROW_CLASS}">
         <label for="subject1">Subject 1</label>
-        <input class="govuk-input" id="s1" value="Maths">
+        <input class="${GOVUK_INPUT_CLASSNAME}" id="s1" value="Maths" />
       </div>
       <div class="${ROW_CLASS}">
         <label for="subject2">Subject 2</label>
-        <input class="govuk-input" id="s2" value="Music">
-        <a id="delete_2" class="${DELETE_BUTTON_CLASSNAME}" href="#">delete subject</a>
+        <input class="${GOVUK_INPUT_CLASSNAME}" id="s2" value="Music" />
+        <a class="${DELETE_BUTTON_CLASSNAME}" href="#">delete subject</a>
       </div>
       <div class="${ROW_CLASS}">
         <label for="subject3">Subject 3</label>
-        <input class="govuk-input" id="s3" value="Geography">
-        <a id="delete_3" class="${DELETE_BUTTON_CLASSNAME}" href="#">delete subject</a>
+        <input class="${GOVUK_INPUT_CLASSNAME}" id="s3" value="Geography" />
+        <a class="${DELETE_BUTTON_CLASSNAME}" href="#">delete subject</a>
       </div>
       <div class="${ROW_CLASS}">
         <label for="subject4">Subject 4</label>
-        <input class="govuk-input" id="s4" value="Economics 101">
-        <a id="delete_4" class="${DELETE_BUTTON_CLASSNAME}" href="#">delete subject</a>
+        <input class="${GOVUK_INPUT_CLASSNAME}" id="s4" value="Economics 101" />
```

Comment:
Might be good to test that the numbers in the value `Economics 101` are not updated when the row is renumbered.